pom.xml

```diff
@@ -84,7 +84,13 @@
         <dependency>
             <groupId>org.jenkins-ci</groupId>
             <artifactId>trilead-ssh2</artifactId>
-            <version>build-217-jenkins-27</version>
+            <version>build-217-jenkins-211.vbb42cae44b18</version>
+            <exclusions>
+                <exclusion>
+                    <groupId>com.google.protobuf</groupId>
+                    <artifactId>protobuf-java</artifactId>
+                </exclusion>
+            </exclusions>
```

Comment:
you should not need to exclude the version here.  the maven dependency will be closest wins, and as you have a direct dependency on `protobuf-java` that would win.

it also means that if/when the upstream library is updated to use a newer protobuf the enforcer [`upperBounds`](https://maven.apache.org/enforcer/enforcer-rules/requireUpperBoundDeps.html) rule would let you know.

```suggestion
```